<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GrafET</class>
 <widget class="QDialog" name="GrafET">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>704</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3" rowstretch="1,1,1,1,20">
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <property name="horizontalSpacing">
    <number>5</number>
   </property>
   <property name="verticalSpacing">
    <number>2</number>
   </property>
   <item row="0" column="1">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Entropy</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Tennengrad</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QDoubleSpinBox" name="E_HP"/>
     </item>
     <item row="1" column="2">
      <widget class="QCheckBox" name="E_DPzobraz">
       <property name="text">
        <string>Show</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QCheckBox" name="E_HPzobraz">
       <property name="text">
        <string>Show</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QDoubleSpinBox" name="E_DP"/>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>lower threshold</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>upper threshold</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="2">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QDoubleSpinBox" name="T_HP">
       <property name="maximum">
        <double>999.990000000000009</double>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QCheckBox" name="T_DPzobraz">
       <property name="text">
        <string>Show</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QCheckBox" name="T_HPzobraz">
       <property name="text">
        <string>Show</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QDoubleSpinBox" name="T_DP">
       <property name="maximum">
        <double>999.990000000000009</double>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>lower threshold</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>upper threshold</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="ohodnoceniEntropyCB">
       <property name="text">
        <string>entropy evaluation</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="ohodnoceniTennenCB">
       <property name="text">
        <string>tennengrad evaluation</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="prvniRozhodCB">
       <property name="text">
        <string>First decision</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="druheRozhodCB">
       <property name="text">
        <string>Second decision</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCheckBox" name="ohodnocKomplet">
       <property name="text">
        <string>Complete evaluation</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="IV1">
       <property name="text">
        <string>Eval. index 1</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="IV4">
       <property name="text">
        <string>Eval. index 4</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="IV5">
       <property name="text">
        <string>Eval. index 5</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCheckBox" name="zobrazGrafE">
       <property name="text">
        <string>Show entropy</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="zobrazGrafT">
       <property name="text">
        <string>Show tennengrad</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="3" rowspan="2">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Save results</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Save thresholds</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0" colspan="4">
    <layout class="QVBoxLayout" name="graphLayout"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
